src/main/java/uk/gov/hmcts/reform/professionalapi/service/impl/OrganisationServiceImpl.java

```diff
@@ -380,6 +381,7 @@ public OrganisationResponse updateOrganisation(
         if (isNotEmpty(savedOrganisation.getPaymentAccounts())
                 && organisationCreationRequest.getStatus().equals("ACTIVE")) {
             updatePaymentAccounts(savedOrganisation.getPaymentAccounts());
+            organisation.setDateApproved(LocalDateTime.now());
```

Comment:
This is not right place to set Org date approved , Org is already persisted couple of lines above will not take any effect. Also need to make sure if there is Org approved request then only need to persists approveDate.
SuperController.updateOrganisationById is right place to add dateapproved field

Reply:
resolved